index.bs

```diff
@@ -127,7 +127,7 @@ When invoked on {{Document}} |doc| with {{USVString}} |requestedOrigin|, the <df
         1. Return.
     1. Let |permissionState| be the result of [=requesting permission to use=] "<a permission><code>top-level-storage-access</code></a>" with |descriptor|.
 
-        NOTE: Note that when requesting permissions and deciding whether to show a prompt, user agents apply implementation-defined behavior to shape the end user experience. Particularly for `top-level-storage-access`, user agents are known to apply custom rules that will grant or deny a permission without showing a prompt.
+        NOTE: Note that when requesting permissions and deciding whether to show a prompt, user agents apply implementation-defined behavior to shape the end user experience. Particularly for `top-level-storage-access`, user agents are known to apply custom rules that will grant or deny a permission without showing a prompt. User agents may also treat the `top-level-storage-access` permission as a "superset" of other permissions, and may grant those other permissions when `top-level-storage-access` permission is granted.
```

Comment:
A few thoughts:

- This doesn't seem placed right. In this step we check for permission for `top-level-storage-access`, we're not trying to determine other permissions.
- It's generally discouraged (sometimes disallowed) to use RFC 2119 terms ("may", in this case) in informative notes.
- I can't really wrap my head around the "superset" concept for the relationship between these two permissions, but maybe that's just me :)

Would it make sense to just add another normative step here with a MAY, something like:

```
1. The user agent may (or should?) grant the "storage-access" permission for origin and doc's origin.
```

Reply:
Thinking about this more maybe this should be just a regular step, it's not clear why it has to be may/should :)

Reply:
PTAL - I've updated the PR to still mandate the `top-level-storage-access`/`storage-access` dual write, but do so by properly integrating into the Permissions spec.